consul_dev_cluster2/scripts/dnsmasq.sh

```diff
@@ -2,10 +2,10 @@
 
 set -x
 
-which redis-cli  2>/dev/null || {
+which dnsmasq  2>/dev/null || {
   export DEBIAN_FRONTEND=noninteractive
   apt-get update
-  apt-get install -y dnsmasq redis-tools 
```

Comment:
This probably makes sense to have it in the legacy app or in a different provision script, like redis-client.sh